2/camille-baldock-captions.srt

```diff
@@ -2166,7 +2166,7 @@ And for each of those cards I have Pips.
 
 498
 00:33:08,480 --> 00:33:11,920
-I can just inaudible
+I can just .....
```

Comment:
That's a lot of dots! I'd prefer to see `I can just...` - that is, no space then 3 dots. I'll update our style-guide to reflect this.